README.md

```diff
@@ -60,6 +60,34 @@ An initial run_list for the node can also be passed to the subcommand. Example u
 
     knife bootstrap windows ssh ec2-50-xx-xx-124.compute-1.amazonaws.com -r 'role[webserver],role[production]' -x Administrator -i ~/.ssh/id_rsa
 
+### knife windows cert generate
+
+Generates a certificate(x509) containing a public / private key pair for WinRM 'SSL' communication.
+The certificate will be generated in three different formats *.pfx, *.der and *.pem.
+The PKCS12(i.e *.pfx) will be added to WinRM Server's Certificate Store.
+The *.der is base64 representaion of *.pfx which may be useful in cloud provider api.
+The *.pem contains public key and can be used at client side.
+This command also displays thumbprint of generated certificate.
+
+    knife windows cert generate --cert-passphrase "strong_passphrase" --domain "cloudapp.net" --output-file "C:\Users\Administrator\server_cert.pfx"
+    # This command will generate certificates at "C:\Users\Administrator\" location with names server_cert.der, server_cert.pfx and server_cert.pem.
```

Comment:
This command works on both Unix and Windows -- we should use an example that works in both cases. Assuming you're using PowerShell, you can use paths such as `~/server_cert.pfx` rather than `c:\users\administrator\server_cert.pfx`.